django-backend/fecfiler/scha_transactions/models.py

```diff
@@ -51,8 +51,8 @@ class SchATransaction(models.Model):
     memo_text_description = models.CharField(null=True, blank=True, max_length=100)
     reference_to_si_or_sl_system_code_that_identifies_the_account = models.CharField(null=True, blank=True, max_length=9)
     transaction_type_identifier = models.CharField(null=True, blank=True, max_length=12)
-    created = models.DateField(auto_now_add=True)
-    updated = models.DateField(auto_now=True)
+    created = models.DateTimeField(auto_now_add=True)
+    updated = models.DateTimeField(auto_now=True)
```

Comment:
Can we add a "deleted" column here as well of type boolean? We'll need to override the model delete with setting this field to "true" in the database table.